flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/FlinkResourceManager.java

```diff
@@ -113,7 +111,8 @@
 	/** The service to find the right leader JobManager (to support high availability) */
 	private final LeaderRetrievalService leaderRetriever;
 
-	/** The currently registered resources */
+	/** The resources which have been registered at the ResourceManager. This indicates the set
+	 * of containers where a TaskManager has been started and registered at a JobManager.  */
```

Comment:
Yes and we should change the comment to not state that they have been registered at the JobManager because currently they are registered at the ResourceManager before JobManager registration.